src/adv.py

```diff
@@ -21,7 +21,7 @@
 earlier adventurers. The only exit is to the south."""),
 }
 
-
+# comment for Git
```

Comment:
so it doesn't look like you got the loop started yet to do any moves. but the rest of your code looks like it should be good. Keep up the good work and get it done today! 💪